hxinsight-extension/src/main/resources/alfresco/module/alfresco-hxinsight-connector-hxinsight-extension/context/service-context.xml

```diff
@@ -174,4 +174,15 @@
     <bean class="org.alfresco.hxi_connector.hxi_extension.rest.api.QuestionFeedbackRelation">
         <constructor-arg name="hxInsightClient" ref="hxInsightClient"/>
     </bean>
+
+    <bean id="acsRepositoryInformation" class="org.alfresco.hxi_connector.hxi_extension.service.AcsRepositoryInformation">
+        <constructor-arg name="descriptorService" ref="DescriptorService"/>
+    </bean>
+    <bean id="hxiApplication" class="org.alfresco.hxi_connector.common.config.properties.Application">
```

Comment:
The bean name (and also property bellow) is confusing for me - I assume it's connector version, not some HxI AI app version

Reply:
Changed - hopefully new names are better suiting and less confusing.